### PR TITLE
Remove Avr boards

### PR DESCRIPTION
**Pull request by dgatf**
Removes remaining code for avr boards

Also the following files/folder from companion may be removed: ~~_burnconfigdialog.*_~~ _fusesdialog.*, er9x_

Should I delete them?

### CODE REVIEW

File: companion/src/wizarddata.cpp [hunk 1]
```diff
@@ -61,27 +61,22 @@ void WizMix::maxMixSwitch(char *name, MixData &mix, int channel, int sw, int wei
...
-      else
-        mix.srcRaw = RawSource(SOURCE_TYPE_STICK, input-1);
+      int channel = settings.getDefaultChannel(input-1);
+      mix.srcRaw = RawSource(SOURCE_TYPE_VIRTUAL_INPUT, channel);
       mix.weight = weight;
     }
     else if (input==FLAPS_INPUT){
       // There ought to be some kind of constants for switches somewhere...
-      maxMixSwitch((char *)"Flaps Up", model.mixData[mixIndex++], channel+1, isHorusOrTaranis ? SWITCH_SA0 :-SWITCH_ELE ,  weight); //Taranis-Horus SA-UP, 9X ELE-UP
-      maxMixSwitch((char *)"Flaps Dn", model.mixData[mixIndex++], channel+1, isHorusOrTaranis ? SWITCH_SA2 : SWITCH_ELE , -weight); //Taranis-Horus SA-DOWN, 9X ELE-DOWN
```
Comment (kilrah) [inline companion/src/wizarddata.cpp:79]: This needs to stay, Sky9X/AR9X/9XR-Pro still fall in the case

File: companion/src/wizarddata.cpp [hunk 2]
```diff
@@ -61,27 +61,22 @@ void WizMix::maxMixSwitch(char *name, MixData &mix, int channel, int sw, int wei
...
+      maxMixSwitch((char *)"Flaps Up", model.mixData[mixIndex++], channel+1, SWITCH_SA0,  weight);
+      maxMixSwitch((char *)"Flaps Dn", model.mixData[mixIndex++], channel+1, SWITCH_SA2, -weight);
 
     }
     else if (input==AIRBRAKES_INPUT){
-      maxMixSwitch((char *)"AirbkOff", model.mixData[mixIndex++], channel+1, isHorusOrTaranis ? SWITCH_SE0 :-SWITCH_RUD , -weight); //Taranis-Horus SE-UP, 9X RUD-UP
-      maxMixSwitch((char *)"AirbkOn",  model.mixData[mixIndex++], channel+1, isHorusOrTaranis ? SWITCH_SE2 : SWITCH_RUD , weight); //Tatanis-Horus SE-DOWN, 9X RUD-DOWN
```
Comment (dgatf) [inline companion/src/wizarddata.cpp:84]: Same as before

File: companion/src/wizarddata.cpp [hunk 3]
```diff
@@ -120,7 +115,7 @@ WizMix::operator ModelData()
         mix.srcRaw = SOURCE_TYPE_MAX;
         mix.weight = -100;
         mix.swtch.type = SWITCH_TYPE_SWITCH;
-        mix.swtch.index = IS_ARM(getCurrentBoard()) ? SWITCH_SF0 : SWITCH_THR;
```
Comment (dgatf) [inline companion/src/wizarddata.cpp:123]: Same as before

File: companion/src/wizarddata.cpp (hunk 1 above)
Comment (dgatf) [reply companion/src/wizarddata.cpp:79]: Absolutely. Also macro IS_ARM was wrong. It should be IS_HORUS_OR_TARANIS. Fixed

Comment (bsongis) [reply companion/src/wizarddata.cpp:79]: or a IS_9X(...) macro?

Comment (dgatf) [reply companion/src/wizarddata.cpp:79]: Yes, clearer. Updated

**Comment by dgatf** [comment]
damn... telemetry tab is lost

File: companion/src/firmwares/opentx/opentxinterface.cpp [hunk 4]
```diff
@@ -593,19 +538,19 @@ int OpenTxFirmware::getCapability(::Capability capability)
...
       return (IS_HORUS_OR_TARANIS(board) ? false : true);
     case ChannelsName:
-      return (IS_ARM(board) ? (HAS_LARGE_LCD(board) ? 6 : 4) : 0);
+      return (HAS_LARGE_LCD(board) ? 6 : 4);
     case HasCvNames:
-      return (IS_ARM(board) ? 1 : 0);
+      return 1;
     case Telemetry:
-      if (IS_2560(board) || IS_ARM(board) || id.contains("frsky") || id.contains("telemetrez"))
+      if (id.contains("frsky") || id.contains("telemetrez"))
```
Comment (kilrah) [inline companion/src/firmwares/opentx/opentxinterface.cpp]: This is where you lose the telemetry panel, IS_ARM() in the first branch should bring everyone there. You can remove frsky and telemetrez, they are obsolete too

Comment (dgatf) [reply companion/src/firmwares/opentx/opentxinterface.cpp]: Yes. Fixed

Maybe TM_HASTELEMETRY, TM_HASOFFSET, TM_HASWSHH is obsolete too? If so, returning 1 and remove from elesewhere would be better also

Comment (kilrah) [reply companion/src/firmwares/opentx/opentxinterface.cpp]: Had a look, TM_HASWSHH is obsolete indeed, TM_HASOFFSET as well, as is the entire TelemetryAnalog class, ui and related code in telemetry.cpp
So yes return 1 and adapt matching code...

Comment (dgatf) [reply companion/src/firmwares/opentx/opentxinterface.cpp]: Ok. Removed _TelemetryAnalog_ class. File _telemetry_analog.ui_ can be deleted

**Pull request merged**